### PR TITLE
XO and SCC clothing mostly.

### PR DESCRIPTION
**Pull request by DreamySkrell**
image

changes:
  - rscadd: "Adds a random suit item for mapping stuff."
  - rscadd: "Adds a cargo radio key pouch to OM's locker."
  - tweak: "Replaces XO's cargo radio key pouch with a security one."
  - rscadd: "Adds XO's white beret and gloves."
  - tweak: "Tweaks the contents of XO's attire locker."
  - tweak: "Moves SCC loadout items to the faction tab."
  - rscadd: "Adds a SCC beret to the loadout."

The quest for drip is eternal.

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

**Comment by EmperorJayJay** [comment]
Drip is everything.

**Review by KingOfThePing — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/clothing/factions/scc.dm [hunk 1]
```diff
@@ -46,6 +46,7 @@
 /obj/item/clothing/head/beret/scc
 	name = "\improper SCC beret"
 	desc = "A corporate beret in the colours of the Stellar Corporate Conglomerate."
+	desc_fluff = "The Stellar Corporate Conglomerate, also known as Chainlink, is a joint alliance between the NanoTrasen Corporation, Hephaestus Industries, Idris Incorporated, Zeng-Hu Pharmaceuticals and Zavodskoi Interstellar to exercise an undisputed economic dominance over the Orion Spur."
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/scc.dm:49]:
```suggestion
	desc_fluff = "The Stellar Corporate Conglomerate, also known as the Chainlink, is a joint alliance between the NanoTrasen, Hephaestus Industries, Idris Incorporated, Zeng-Hu Pharmaceuticals, and Zavodskoi Interstellar corporations to exercise an undisputed economic dominance over the Orion Spur."
```

**Comment by SleepyGemmy** [comment]
a

**Review by SleepyGemmy — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**